src/main/java/io/parallec/core/bean/ssh/SshMeta.java

```diff
@@ -58,6 +58,12 @@
     // ssh
     private int sshConnectionTimeoutMillis;
 
+    /** The ssh jump host name */
+    private String sshJumpHost;
+
+    /** The ssh jump host meta */
+    private SshMeta sshJumpHostMeta;
```

Comment:
Have some concerns on `sshJumpHostMeta` being type `SshMeta`.  SshMeta has around 10 attributes and I believe we just use username port and password , and this recursive pattern looks confusing to people.  

Could we just add the attribute directly to SshMeta, or create a new class just to wrap the attributes used?